components/login/login_controller/login_controller.jsx

```diff
@@ -244,20 +264,20 @@ export default class LoginController extends React.Component {
     }
 
     createCustomLogin() {
-        if (global.window.mm_license.IsLicensed === 'true' &&
-                global.window.mm_license.CustomBrand === 'true' &&
-                global.window.mm_config.EnableCustomBrand === 'true') {
-            const text = global.window.mm_config.CustomBrandText || '';
+        if (this.props.isLicensed &&
+                this.props.customBrand &&
+                this.props.enableCustomBrand) {
+            const text = this.props.customBrandText || '';
             const formattedText = TextFormatting.formatText(text);
 
             return (
                 <div>
                     <img
                         src={Client4.getBrandImageUrl(0)}
                     />
-                    <p>
+                    <div>
                         {messageHtmlToComponent(formattedText, false, {mentions: false})}
-                    </p>
```

Comment:
This looks like no related with the commit at all, fixes something or is accidentally introduced?

Reply:
Yeah, React was throwing a warning about nested paragraph tags, which this fixed without any visual changes. I'll pull this out as a separate PR, though, to avoid cluttering it in with this one.

Reply:
I think is not needed a new PR for this. If you have checked there is not visual changes, is enough for me. Is common to include some polishing changes not related in the same PR. And probably the overhead of a new PR and review process, doesn't worth it.